src/config/mod.rs

```diff
@@ -53,14 +54,13 @@ pub struct Config {
 }
 
 #[derive(Clone, Debug, Default, Deserialize, Serialize)]
+#[serde(default)]
 pub struct GlobalOptions {
     #[serde(default = "default_data_dir")]
     pub data_dir: Option<PathBuf>,
-    #[serde(
-        skip_serializing_if = "crate::serde::skip_serializing_if_default",
-        default
```

Comment:
Was dropping the `default` here intentional?

Reply:
Yes because of adding the `default` at the struct level.

Reply:
Doh, missed that. 👍